src/main.cpp

```diff
@@ -255,10 +254,10 @@ void loop()
     {
       converter.readRegistryValues(buff); //process all values from the register
       updateValues(registryIDs[i]);       //send them in mqtt
-      waitLoop(500);//wait .5sec between registries
+      //waitLoop(500);//wait .5sec between registries
     }
   }
   sendValues();//Send the full json message
-  mqttSerial.printf("Done. Waiting %d sec...\n", FREQUENCY / 1000);
+  mqttSerial.printf("Done. Waiting %d sec...", FREQUENCY / 1000);
```

Comment:
The newline lead to serious problems with some MQTT listeners. So removing for compatibility.